components/main_menu/main_menu.jsx

```diff
@@ -119,6 +120,15 @@ class MainMenu extends React.PureComponent {
         }
     }
 
+    shouldShowUpgradeModal = () => {
+        const {subscriptionStats, isCloud} = this.props;
+
+        if (subscriptionStats?.is_paid_tier === 'true') { // eslint-disable-line camelcase
+            return false;
+        }
+        return isCloud && subscriptionStats?.remaining_seats <= 0;
+    }
+
```

Comment:
I believe this is no longer required given that Pablo introduced this check within the `invitePeopleModal`.  @pablovelezvidal could you confirm if that is in master?

Reply:
yes, it was recently merged, so merging master should make the trick.